docs/lib/auth/fragments/native_common/check_authentication_state/common.md

```diff
@@ -0,0 +1,6 @@
+Use the following method to check if the current user is authenticated.   This is the recommended approach for determining whether you should display a sign in screen when your application starts.
+
+<amplify-callout>
+`fetchAuthSession().isSignedIn()` returns true even after the session is expired. It's necessary to check whether refresh token and AuthSession is still valid or not before calling `fetchAuthSession().isSignedIn()`.
```

Comment:
- In what use case do I use ``fetchAuthSession().isSignedIn()`? When do I not care whether the session is expired or not?
- as a reader, i don't know what a refresh token is, perhaps "A valid refresh token indicates that the session is still valid".
- what is AuthSession? is it the authentication session? why do we have to check both the AuthSession and the refresh token? why do I have to check `fetchAuthSession().isSignedIn()` afterwards?